console-subscriber/src/aggregator.rs

```diff
@@ -67,6 +70,12 @@ pub(crate) struct Flush {
     pub(crate) triggered: AtomicBool,
 }
 
+#[derive(Debug)]
+enum Temporality {
```

Comment:
cute name :)

assuming this is an enum because we might want to add replays or other forms of time travel?